Assets/Scripts/Game/UserInterfaceWindows/DaggerfallShortcut.cs

```diff
@@ -0,0 +1,295 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DaggerfallWorkshop.Game.UserInterfaceWindows
```

Comment:
this is not a window so should be in a different namespace IMO

Reply:
Okay, I think it's barely above InputManager, and could be used outside of the UI to detect key combos, so I'm moving it one level up

Reply:
i would suggest move across to Game/UserInterface along with all the UI building blocks

Reply:
Yes, I was thinking of HotkeySequence.cs
Indeed DaggerfallShortcuts is more related to UI, will do